src/compressor/zlib/CMakeLists.txt

```diff
@@ -50,6 +50,7 @@ endif(HAVE_INTEL_SSE4_1 AND HAVE_BETTER_YASM_ELF64 AND (NOT APPLE))
 
 add_library(ceph_zlib SHARED ${zlib_sources})
 target_link_libraries(ceph_zlib ZLIB::ZLIB)
+target_link_libraries(ceph_zlib compressor)
```

Comment:
@qwren could you address @tj90241 's comment and update `doc/radosgw/qat-accel.rst` accordingly?

Reply:
Sure.